examples/NationalInstruments/highlights/example-clicking.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
This example isn't loading correctly for me, it looks like the URLs for the flot resources are wrong.

Reply:
Fixed